Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -208,10 +216,12 @@ public struct PackageBuilder {
     }
 
     /// Returns path to all the items in a directory.
+    /// FIXME: This is generic functionality, and should move to FileSystem.
```

Comment:
A good way to think of it is as a stack of file systems.  A `RerootedFileSystemView` just means that you can pretend that everything exists at the top level of the file system, but other than that it isn't any different from any other file system.